packages/toggle/README.md

```diff
@@ -83,3 +83,89 @@ return (
 For the Toggle to be accessible to screen readers, you **must** pass either [`aria-label`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute) or [`aria-labelledby`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute) to Toggle. Please note, if this is a part of a form, this is in addition to using `htmlFor` to associate a label with the Toggle. You will see TypeScript and console errors if this isn't done.
 
 Please reach out if you would like further guidance on how to programmatically associate text with the Toggle component.
+
+## getLGToggleUtils()
```

Comment:
Do y'all think this is a good format?

Reply:
@bruugey, do you think this is a good format for the README? If it is, then I can update `TextArea` and `TextInput` to match.

Reply:
I like it! What do you think @TheSonOfThomp

Reply:
I wonder if the naming convention should refer to "test utils"? This name feels like utilities that will help with using the toggle

Reply:
Otherwise, I think I'm good with the format. We _could_ consider a separate `README` in the folder that contains the test utils
Would we want to document these on `.design` eventually?

Reply:
so, `getLGToggleTestUtils`? I'm ok with that

Reply:
Do you think putting the `README` in the test utils folder will be hard to find?

Reply:
@bruugey thoughts on changing the util names to include `TestUtils`? 
`getLGToggleUtils` => `getLGToggleTestUtils`